t/api/02-iso-download.t

```diff
@@ -298,4 +298,28 @@ subtest 'download task only blocks the related job when test suites have differe
     is scalar(@{$gru_dep_tasks->{$gru_task_ids[0]}}), 3, 'one download task was created and it blocked 3 jobs';
 };
 
+subtest 'placeholder expansions work with _URL-derived settings' => sub {
+    $test_suites->find({name => 'kde'})->settings->create({key => 'FOOBAR', value => '%ISO%'});
+    $rsp
+      = schedule_iso({%params, ISO_URL => 'http://localhost/openSUSE-13.1-DVD-i586-Build0091-Media.iso', TEST => 'kde'},
+        200);
```

Comment:
style nitpick as these three lines are not necessarily nicely readable:

```perl
my %new_params = {%params, ISO_URL => 'http://localhost/openSUSE-13.1-DVD-i586-Build0091-Media.iso', TEST => 'kde'};
$rsp = schedule_iso(\%new_params, 200);
```

you can also define that outside the subtest so that you can reuse it later on